### ISSUE TITLE
Add a TL;DR chapter

### ISSUE THREAD

**Issue by kaustavdm** [MozillaIndia/openbook#3]
This chapter will be present before the full structure. It will contain a summarized version of the entire proposal.

Current source is at [manuscript/tldr.md](https://github.com/MozillaIndia/openbook/blob/master/manuscript/tldr.md).

**Comment by asdofindia** [comment]
How do we summarize geographic teams in the tl;dr? I think a good one liner would be "geographic teams (auto-generated teams that get together when the scope of an activity is restricted to a geographic location)"

**Comment by kaustavdm** [comment]
That's a good start. But, the other purpose of geographic teams is to aid discovery at a local level. As in, it answers the question, "Who is near me?". How do we add that in?

**Comment by asdofindia** [comment]
Geographic teams (people who are at the same place and represent Mozilla India at that place)

**Comment by kaustavdm** [comment]
Nope. This one does not highlight the transient nature of membership of geographic teams (which is the biggest deal there). Probably a mixture of the two suggestions would work.

**Comment by asdofindia** [comment]
Mix those?

**Comment by kaustavdm** [comment]
How about:

> Geographic groups help in discovering and connecting Mozillians in a particular region. These groups get together when the scope of an activity is restricted to a geographic location. Membership to geographic groups happen by virtue of one's presence in that region.

**Comment by kaustavdm** [comment]
@asdofindia care to submit a patch?

**Comment by kaustavdm** [comment]
@asdofindia Figure out a way of adding a line about facilitators somewhere?
